### PR TITLE
♻️ Base `connect_hub()` off of edge function

### PR DESCRIPTION
**Pull request by Koncopd**

### CODE REVIEW

File: lamindb_setup/core/_settings_instance.py [hunk 1]
```diff
@@ -56,6 +56,8 @@ def __init__(
         schema: str | None = None,  # comma-separated string of schema names
         git_repo: str | None = None,  # a git repo URL
         is_on_hub: bool | None = None,  # initialized from hub
+        api_url: str | None = None,
+        schema_id: UUID | None = None,
```
Comment (Koncopd) [inline lamindb_setup/core/_settings_instance.py:60]: @falexwolf not sure. I have to add them here for saving the instance settings. But i am not sure if they should be here even as just private variables.

Comment (falexwolf) [reply lamindb_setup/core/_settings_instance.py:60]: I think this is OK, Sergei!

**Comment by falexwolf** [comment]
@Koncopd, can we merge this one?

**Comment by Koncopd** [comment]
One more thing and i will merge.

**Pull request merged**